src/backend/renderer/multigpu/gbm.rs

```diff
@@ -129,6 +143,8 @@ impl<R: From<GlesRenderer> + Renderer<Error = GlesError>> GraphicsApi for GbmGle
     type Error = Error;
 
     fn enumerate(&self, list: &mut Vec<Self::Device>) -> Result<(), Self::Error> {
+        self.wants_enumeration.store(true, Ordering::SeqCst);
```

Comment:
This probably was meant to clear `wants_enumeration` by storing `false`?

Reply:
Yup. Copy&Paste mistake.